src/test/java/org/folio/search/support/base/BaseIntegrationTest.java

```diff
@@ -197,6 +198,8 @@ protected static ResultActions attemptSearch(
     doIfNotNull(limit, value -> requestBuilder.queryParam("limit", String.valueOf(value)));
     doIfNotNull(offset, value -> requestBuilder.queryParam("offset", String.valueOf(value)));
     doIfNotNull(expandAll, value -> requestBuilder.queryParam("expandAll", String.valueOf(value)));
+    doIfNotNull(limit, value -> requestBuilder.queryParam("tenantId", CENTRAL_TENANT_ID));
+    doIfNotNull(limit, value -> requestBuilder.queryParam("sortBy", "name"));
```

Comment:
I suppose this method is used in multiple places, not only for locations so not a good idea to always add those parameters. And conditional on limit is strange to see here

Reply:
this was added for testing purposes, should be deleted